neo/Network/RPC/Helper.cs

```diff
@@ -0,0 +1,108 @@
+using Neo.Cryptography.ECC;
+using Neo.SmartContract;
+using Neo.Wallets;
+using System;
+using System.Numerics;
+
+namespace Neo.Network.RPC
+{
+    public static class Helper
```

Comment:
Now the only thing that I don't like is this file. If it is for public use, this is not the right place. Maybe it should be moved to `Neo.Utils`. Otherwise, it should be `internal`. I think some of these functions are useful and can be public used. `ToBigInteger()` can be `internal`. `ToScriptHash(this KeyPair key)` can be removed.

Reply:
Hi @erikzhang 
I have removed `ToScriptHash(this KeyPair key)` and make `ToBigInteger()` as `internal`.
Will we add the `Neo.Utils` namespace in this PR?

Reply:
Or shall we add `ToUInt160()` and `ToKeyPair()` to `Neo.Helper`? @erikzhang